packages/lib/storage/service.ts

```diff
@@ -211,8 +212,15 @@ export const getS3UploadSignedUrl = async (
   environmentId: string,
   isBiggerFileUploadAllowed: boolean = false
 ) => {
-  const maxSize = isBiggerFileUploadAllowed ? MAX_SIZES.big : MAX_SIZES.standard;
-  const postConditions: PresignedPostOptions["Conditions"] = [["content-length-range", 0, maxSize]];
+  const maxSize = IS_FORMBRICKS_CLOUD
```

Comment:
Not super important, but we can also create a function for calculating the max size in `utils.ts`. What do you think ? 🤔

Reply:
I tried this, but constants are not available in utils, we might need to pass many arguments just to replace one ternary, we can look at this later.